src/pages/HomeScreen.tsx

```diff
@@ -202,7 +212,7 @@ export class HomeScreen extends BaseScreen {
 
     if (barcodeResult.status === 'OK') {
       CachedBarcodeResult.update(barcodeResult);
-      CachedBarcodeResult.imageUri = 'data:image/png;base64,' + response.data;
+      CachedBarcodeResult.imageUri = 'data:image/png;base64,' + selectedImageuri;
```

Comment:
This looks wrong, since `data:image/png;base64,` is used to anticipate BASE64 data, not an imageUri.

I would remove that and just pass the URI, and test if it's working properly